sklearn/feature_extraction/text.py

```diff
@@ -1430,6 +1431,9 @@ class TfidfTransformer(TransformerMixin, BaseEstimator):
     .. [MRS2008] C.D. Manning, P. Raghavan and H. Schütze  (2008).
                    Introduction to Information Retrieval. Cambridge University
                    Press, pp. 118-120.
+
+    .. versionadded:: 0.19
```

Comment:
Firstly, this should not be in the References section.

Secondly, TfidfTransformer was not added as late as version 0.19. It was added in version 0.5 as far as I can tell!

How did you decide it was added in 0.19, and is it possible you could check for similar errors.